src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
I understand your desire to be standards-compliant here but for the xbox app in particular, this would require us to set tabIndex to -1 for every div that's used. I don't think that's a particularly beneficial thing to do even in the name of being standards-compliant. How about we make arcade-machine select things that have positive tabIndex (that is, excluding 0)?